### PR TITLE
FAQs

### PR DESCRIPTION
**Pull request by palmenhq**

### CODE REVIEW

File: src/pages/OfferNew/FaqSection.tsx [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+  height: 2rem;
+  fill: ${colorsV2.black};
+  transform: rotateY(180deg);
+`
+
+const SectionWrapper = styled.div`
+  padding: 8rem 2rem 12rem 2rem;
+  margin-top: -10px;
+  background: ${colorsV2.black};
+  color: #fff;
```
Comment (hedvigoscar) [inline src/pages/OfferNew/FaqSection.tsx]:
```suggestion
  color: ${colorsV2.WHITE};
```

File: src/pages/OfferNew/FaqSection.tsx [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+  fill: #fff;
+`
+
+interface AccordionProps {
+  id: string
+  headline: string
+  body: string
+}
+
+export const Accordion: React.FunctionComponent<AccordionProps> = ({
```
Comment (hedvigoscar) [inline src/pages/OfferNew/FaqSection.tsx]:
```suggestion
export const Accordion: React.FC<AccordionProps> = ({
```

File: src/pages/OfferNew/FaqSection.tsx [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+}) => {
+  const [isOpen, setIsOpen] = React.useState(false)
+  const toggleIsOpen = () => setIsOpen(!isOpen)
+
+  return (
+    <AccordionWrapper>
+      <AccordionHeadline>
+        <ExpandToggler onClick={toggleIsOpen}>
+          <AccordionHeadlineContent>{headline}</AccordionHeadlineContent>
+          <ExpanderIcon viewBox="0 0 357 357" isOpen={isOpen}>
```
Comment (hedvigoscar) [inline src/pages/OfferNew/FaqSection.tsx]: Should this SVG be defined inlined like this? Perhaps we could extract it.

Not a biggie though 👍

Comment (palmenhq) [reply src/pages/OfferNew/FaqSection.tsx]: i think its fine for one that is this small no?

**Pull request merged**